### DISCUSSION TITLE
New on ARCore

### DISCUSSION THREAD

**Discussion by kelvenchin** [sceneview/sceneform-android#188] [Q&A]
I'm new in AR, anyone can give some tips or video link on how to start a simple AR project from nothing to complete project ?

**Comment by ThomasGorisse** [comment]
Have a look at this article: [Create an Android AR app](https://medium.com/make-an-android-ar-augmented-reality-app-for/make-an-ar-augmented-reality-app-for-android-in-2022-48a1711562bb)
It's not really a tutorial but explains the actual Android ways to go you have.

If you choose this solution, depending on what you want to achieve, have a look at the [samples](https://github.com/ThomasGorisse/SceneformMaintained/tree/master/samples).
[glTF 3D model display sample](https://github.com/ThomasGorisse/SceneformMaintained/tree/master/samples/gltf) is a good start

**Reply by kelvenchin** [reply]
Because I'm working on my assignment project for implementing AR into my android apps develop by android studio, is it workable for using the sceneform maintain right now ?

**Reply by kelvenchin** [reply]
I have tried to run your sample project glTF 3D model but it can't be run successfully...

**Reply by RGregat** [reply]
Can you give a few more information about what fails? A Stacktrace, picture, and so on.

**Reply by kelvenchin** [reply]
After I install google play services AR on my emulator, it works for me, the object can be place out when click on anchornode. But the logcat there always looping the same error like

"2021-10-28 16:15:01.553 5943-5943/com.google.ar.sceneform.samples.gltf E/native: E1028 08:15:01.553878    5943 hit_test.cc:428] generic::internal: No point hit.
2021-10-28 16:15:01.555 5943-5943/com.google.ar.sceneform.samples.gltf E/eglCodecCommon: glUtilsParamSize: unknow param 0x00008919
2021-10-28 16:15:01.555 5943-5943/com.google.ar.sceneform.samples.gltf E/eglCodecCommon: glUtilsParamSize: unknow param 0x00008919"

**Reply by kelvenchin** [reply]
Is it normal ?

**Comment by kelvenchin** [comment]
Can I ask when I create a new project I only need to add dependencies ? Or I need to manually put in the sceneform, ux and core folders ?

**Reply by RGregat** [reply]
The dependency is enough.

**Reply by kelvenchin** [reply]
the sceneform 1.16.0 also no need manually download and place into project folder ?

**Reply by kelvenchin** [reply]
and the snapshot dependencies is it neccessary to put it in ?

**Comment by kelvenchin** [comment]
`import com.google.ar.sceneform.Sceneform;` this import the Sceneform word is in red colour , is it something changes in this ?

**Reply by RGregat** [reply]
That is the wrong dependency, pls take a look to the README of this repository [Dependency](https://github.com/ThomasGorisse/SceneformMaintained#dependencies)

**Reply by kelvenchin** [reply]
But the dependencies is take from the README dependency, is it wrong ?

**Comment by kelvenchin** [comment]
`dependencies{
implementation("com.gorisse.thomas.sceneform:sceneform:version")
implementation 'com.github.thomasgorisse.sceneformmaintained:sceneform:-SNAPSHOT'
}
allprojects {
    repositories {
        mavenCentral()
        maven { url 'https://jitpack.io' }
    }
}
`
I put in these dependencies and try to put in the code take from README in java file but there is some errors 

`import com.google.ar.sceneform.Sceneform;
BaseArFragment.OnSessionConfigurationListener,
        ArFragment.OnViewCreatedListener`

This is the part of the code that in error

**Comment by ThomasGorisse** [comment]
@kelvenchin We are really sorry but **can you please learn Android development before asking questions here?**

They are great tutorials for that but we won't teach you how to make your first Android app here.
